tableauserverclient/server/endpoint/endpoint.py

```diff
@@ -15,6 +16,16 @@ def _check_status(server_response):
         if server_response.status_code not in Success_codes:
             raise ServerResponseError.from_response(server_response.content)
 
+    def get_unauthenticated_request(self, url, request_object=None):
+        if request_object is not None:
+            url = request_object.apply_query_params(url)
+        server_response = self.parent_srv.session.get(url, **self.parent_srv.http_options)
+        self._check_status(server_response)
+        if server_response.encoding:
```

Comment:
Question -- why check for encoding? Will it ever not be there?

Does this mean if there isn't an encoding there is nothing to log?